phtc/templates/quizblock/quizblock.html

```diff
@@ -5,15 +5,19 @@
   {% for question in block.question_set.all %}
     {% include 'quizblock/question.html' %}
   {% endfor %}
+  <script>
+  jQuery(document).ready(function(){
+	  jQuery("button.reading-exercise").on('click', function(evt){
+		  jQuery(this).parent().parent().find("div.question_response").css('display','block');
```

Comment:
I think that `jQuery(this).closest('div.question_response')` would be more flexible than `jQuery(this).parent().parent().find('div.question_response')` in case a div is added or removed in the future.